### DISCUSSION TITLE
NoiseModelling docker image

### DISCUSSION THREAD

**Discussion by tomasanda** [Universite-Gustave-Eiffel/NoiseModelling#307] [Show and tell]
Hi, 

I have recently created a docker image for NoiseModelling. Feel free to try it out, any feedback will be greatly appreciated.

Docker repositories are shown below:
- [Github repository](https://github.com/tomasanda/docker-noisemodelling)
- [Dockerhub repository](https://hub.docker.com/r/tomasanda/noisemodelling)

Best regards
Tom

**Comment by pierromond** [comment]
Thank you @tomasanda! I just tried it, it works very well  !

We are currently having a discussion in the team to bring a release of NoiseModelling that won't force users to go through the installation of the JAVA environment variable (many users are stuck at this stage as #308 ). Docker installation may seem like a solution, but unfortunately setting up docker for an inexperienced user is not much easier.

Perhaps you have an idea of your own?

**Reply by Nitnelav** [reply]
The Java JRE install stuff should be fixed with a Launch4j exe no ? so https://github.com/Ifsttar/NoiseModelling/issues/291 ?

**Reply by pierromond** [reply]
We postponed this solution because Launch4j finally is not adapted "we do some manipulations (arguments for the VM, export of variables ...)". Nobody reacted about the solution you proposed "Inno Setup". I will speak again with Nicolas and Sylvain about it. We will see. Do you think you could also try @Nitnelav ?

**Reply by tomasanda** [reply]
Maybe we could create a very simple step-by-step guide which will include downloading the Docker Desktop and only the basic commands for running the NM library.

**Reply by pierromond** [reply]
Finally @Nitnelav did an exe file... we think it is the best for a "standard windows user" : https://github.com/Ifsttar/NoiseModelling/releases/tag/v3.3.2

But maybe we should add sthg in the documentation for Docker, it could interest someone. If you have any idea @tomasanda ?

**Reply by pierromond** [reply]
Here a proposition : https://noisemodelling.readthedocs.io/en/latest/Get_Started_Tutorial.html
